pkg/errinsrc/stack.go

```diff
@@ -4,6 +4,8 @@ import (
 	"fmt"
 	"runtime"
 	"strings"
+
+	"github.com/pkg/errors"
```

Comment:
Is this needed? Yet another error library dependency

Reply:
This isn't a new library in the code base (we already used it, as do some of our dependencies), what it is is a semi-standard interface / struct type for different error libraries to use when they want to expose stack trace information.